src/k8s-engine/hyperkit.ts

```diff
@@ -121,16 +135,20 @@ export default class HyperkitBackend extends events.EventEmitter implements K8s.
     return this.dockerMachineConfig.then(v => v?.Driver.Memory ?? 0);
   }
 
+  protected get hyperkitArgs(): { driver: string, defaultArgs: string[] } {
+    return {
+      driver:      resources.executable('docker-machine-driver-hyperkit'),
+      defaultArgs: ['--storage-path', path.join(paths.state(), 'driver')],
```

Comment:
I would call the lowest level `machine` instead of `driver`. That matches `docker-machine` storing its config under `~/.docker/machine`. Alternatively I would call it `hyperkit`, but `driver` feels weird to me.